src/base.jl

```diff
@@ -50,8 +50,8 @@ end
 @inline MPS(A::AbstractArray, ::Val{:right}, Dcut::Int, args...) = _left_sweep_SVD(MPS, A, Dcut, args...)
 @inline MPS(A::AbstractArray, ::Val{:left}, Dcut::Int, args...) = _right_sweep_SVD(MPS, A, Dcut, args...)
 
-@inline Base.dropdims(ψ::MPS, i::Int) = (dropdims(A, dims=i) for A ∈ ψ)
-@inline Base.dropdims(ψ::MPS) = Base.dropdims(ψ, 2)
+@inline Base.dropdims(ψ::AbstractMPS, i::Int) = (dropdims(A, dims=i) for A ∈ ψ)
+@inline Base.dropdims(ψ::AbstractMPS) = Base.dropdims(ψ, 2)
```

Comment:
This shouldn't be `dropdims`, because the default behaviour is confusing and inconsistent with `Base` method. Furthermore, second method is not needed, as `i=2` can be set as default in the first one.

Reply:
Done.